rts/Map/SMF/ROAM/RoamMeshDrawer.cpp

```diff
@@ -188,17 +195,81 @@ void CRoamMeshDrawer::Update()
 
 	bool shadowPass = (cam->GetCamType() == CCamera::CAMTYPE_SHADOW);
 	bool tesselMesh = forceNextTesselation[shadowPass];
+	bool fullRetesselate = false;
 
 	auto& patches = patchMeshGrid[shadowPass];
 	auto& pvflags = patchVisFlags[shadowPass];
 
 	Patch::UpdateVisibility(cam, patches, numPatchesX);
 
-#define RETESSELLATE_MODE 1
+    int numPatchesVisible = 0;
```

Comment:
Maybe the various "number of things" could be `size_t`?
```suggestion
    size_t numPatchesVisible = 0;
```